examples/embedded/src/main/java/org/eclipse/jetty/embedded/ManyHandlers.java

```diff
@@ -115,34 +113,21 @@ public static void main( String[] args ) throws Exception
         HandlerWrapper wrapper = new WelcomeWrapHandler();
         Handler hello = new HelloHandler();
         Handler dft = new DefaultHandler();
-        RequestLogHandler requestLog = new RequestLogHandler();
 
         // configure request logging
         File requestLogFile = File.createTempFile("demo", "log");
         NCSARequestLog ncsaLog = new NCSARequestLog(
                 requestLogFile.getAbsolutePath());
-        requestLog.setRequestLog(ncsaLog);
+        server.setRequestLog(ncsaLog);
 
         // create the handler collections
         HandlerCollection handlers = new HandlerCollection();
         HandlerList list = new HandlerList();
 
         // link them all together
         wrapper.setHandler(hello);
-        list.setHandlers(new Handler[] { param, new GzipHandler(), dft });
-        handlers.setHandlers(new Handler[] { list, requestLog });
-
-        // Handler tree looks like the following
-        // <pre>
-        // Server
-        // + HandlerCollection
-        // . + HandlerList
-        // . | + param (ParamHandler)
-        // . | + wrapper (WelcomeWrapHandler)
-        // . | | \ hello (HelloHandler)
-        // . | \ dft (DefaultHandler)
-        // . \ requestLog (RequestLogHandler)
-        // </pre>
+        list.setHandlers(new Handler[] { param, new GzipHandler() });
+        handlers.setHandlers(new Handler[] { list, dft });
```

Comment:
Do we support having `RequestLogHandler` at the end of the Handler tree still?

Reply:
RequestLogHandler is no longer the preferred way to do RequestLog. Instead we have Server.setRequestLog, which can handle all the various places that a request might fail (and need to be logged) prior to a request even being created and given to a handler.     The only usage for RequestLogHandler now is if you wish to have different RequestLogs for different contexts/handlers.   I'll make sure the javadoc says this clearly.
Also note that now it should be the first handler in a handler list rather than the last (so it can log exceptions etc.).